backend/audit/test_schemas.py

```diff
@@ -524,13 +524,12 @@ class FederalAwardsSchemaValidityTest(SimpleTestCase):
         "FederalAwardsCases"
     ]
 
-    def test_schema(self):
+    def test_multiple_files(self):
         """Try to test FederalAwards first."""
         schema = self.FEDERAL_AWARDS_SCHEMA
-
-        in_flight_file = FEDERAL_AWARDS_TEST_FILE
-        in_flight = json.loads(in_flight_file.read_text(encoding="utf-8"))
-        validate(in_flight, schema)
+        for in_flight_file in FEDERAL_AWARDS_TEST_FILE:
```

Comment:
Isn't `FEDERAL_AWARDS_TEST_FILE` a JSON file (and therefore not something you can iterate over)?